src/main/webapp/dashboard/controller/Master.controller.js

```diff
@@ -66,10 +68,65 @@ sap.ui.define(
           oBinding.sort(oSorter);
         },
 
+        onChange: function(){
+           const oView = this.getView();
+           const aSelectedItems = oView.byId("runtimeConfigTable").getSelectedItems();
+           if(aSelectedItems.length == 1){
+             const me = this;
+             if (!this.byId("changeDialog")) {
+                       // load asynchronous XML fragment
+                       Fragment.load({
+                         id: oView.getId(),
+                         name: "sap.f.ShellBarWithFlexibleColumnLayout.view.ChangeDialog",
+                         controller: this,
+                       }).then(function (oDialog) {
+                         oView.addDependent(oDialog);
+                         me._openChangeDialog(oDialog);
+                       });
+                     } else {
+                       this._openChangeDialog(this.byId("changeDialog"));
+                     }
+           }
+           else{
+            if(aSelectedItems.length == 0){
```

Comment:
Please replace loose equality ("==") with strict equality ("===") for improved type checking.